File: C++ Fundamentals/Basic Syntax/Print and Sum/Print and Sum.cpp
#include <iostream>
using namespace std;

int main()
{
    int start, end;
    cin >> start >> end;

    int sum = 0;
    for (int i = start; i <= end; i++) {
        sum += i;
        cout << i << ' ';
    }

    cout << endl;
    cout << "Sum: " << sum << endl;
}

File: JS Front-End/JS Syntax Fundamentals/Exercises/print-and-sum.js
function solve(start, end) {
    let output = "";
    let sum = 0;
    for (let i = start; i <= end; i++) {
        output += i + " ";
        sum += i;
    }

    console.log(output);
    console.log(`Sum: ${sum}`);
}

solve(5, 10);
solve(0, 26);
solve(50, 60);
